src/suites/cts/command_buffer/copies.spec.ts

```diff
@@ -109,3 +109,67 @@ g.test('b2t2t2b', async t => {
 
   t.expectContents(dst, data);
 });
+
+g.test('imageBitmapToTexture', async t => {
```

Comment:
Let's be more specific about what this test tests: `'from image element'` (imageBitmapToTexture is implicit if moved to copyImageBitmapToTexture.spec.ts).

Reply:
Done.